libs/drape/glyph_manager.cpp

```diff
@@ -246,7 +246,7 @@ FreetypeError constexpr g_FT_Errors[] =
         auto const & metrics = m_fontFace->glyph->metrics;
         auto const xOffset = (currPos.x_offset + static_cast<int32_t>(metrics.horiBearingX)) >> 6;
         // The original Drape code expects a bottom, not a top offset in its calculations.
-        auto const yOffset = (currPos.y_offset + static_cast<int32_t>(metrics.horiBearingY) - metrics.height) >> 6;
+        auto const yOffset = static_cast<int32_t>((currPos.y_offset + metrics.horiBearingY - metrics.height) >> 6);
```

Comment:
xOffset should be the same?

Reply:
Ok, it was already int32_t, type of metrics.height is different.